inst/qml/jaspPlotBuilder.qml

```diff
@@ -0,0 +1,2974 @@
+                        info: qsTr("Select the variable for data grouping, which will also determine the coloring..")
+                        onCountChanged: {
+                            if (count > 0) {
+                                colorByVariableX.checked = false;
+                                colorByVariableY.checked = false;
+                            }
+                        }
+                    }
+                    Group {
+                        title: qsTr("Or color by")
+                        columns: 2
+                        CheckBox {
+                            name: "colorByVariableX"
+                            label: qsTr("X variable")
+                            id: colorByVariableX
+                            enabled: variableXPlotBuilder.count > 0 && variableColorPlotBuilder.count === 0
+                            onCheckedChanged: {
+                                if (checked && variableColorPlotBuilder.count > 0) {
+                                    checked = false;
+                                }
+                                if (checked) {
+                                    colorByVariableY.checked = false;
+                                }
+                            }
+                        }
+                        CheckBox {
+                            name: "colorByVariableY"
+                            label: qsTr("Y variable")
+                            id: colorByVariableY
+                            enabled: variableYPlotBuilder.count > 0 && variableColorPlotBuilder.count === 0
+                            onCheckedChanged: {
+                                if (checked && variableColorPlotBuilder.count > 0) {
+                                    checked = false;
+                                }
+                                if (checked) {
+                                    colorByVariableX.checked = false;
+                                }
+                            }
+                        }
+                    }
+                    AssignedVariablesList {
+                        name: "columnsvariableSplitPlotBuilder"
+                        title: qsTr("Split by (columns)")
+                        id: columnsvariableSplitPlotBuilder
+                        allowedColumns: ["ordinal", "nominal"]
+                        singleVariable: true
+                        info: qsTr("You can choose a variable to split the plots into columns.")
+                    }
+
+                    AssignedVariablesList {
+                        name: "rowsvariableSplitPlotBuilder"
+                        title: qsTr("Split by (rows)")
+                        id: rowsvariableSplitPlotBuilder
+                        allowedColumns: ["ordinal", "nominal"]
+                        singleVariable: true
+                        info: qsTr("You can choose a variable to split the plots into rows.")
+                    }
+
+                    Group {
```

Comment:
I would also move this to the `Plot style` section.
It will make the input interface a bit cleaner and easier to navigate.